### DISCUSSION TITLE
help with fixing some code.

### DISCUSSION THREAD

**Discussion by glowsoony** [KadeArchive/Kade-Engine#2747] [Q&A]
![image](https://user-images.githubusercontent.com/84847356/144728716-a1761cfa-0d86-4095-84b9-b2572e1d586b.png)
everytime i try to run lime test windows it says 
![image](https://user-images.githubusercontent.com/84847356/144728722-10399dbf-c44e-473d-b9e8-938be5516d4a.png)
help. i made the code.

**Comment by libracrumbs** [comment]
I really do not know why that happens, but try to make it like this:

```
}
else
{
```

instead of this:

`}else{
`
If that does not work, i do not know what you should do, but try to modify the code till it works. That's how i fixed it.


_also cool mod but pls change the color scheme it's weird_
